### PR TITLE
Update error message for ImportErrors

### PR DESCRIPTION
**Pull request by KristinaDudnyk**
## Closes: [#309](https://github.com/RaspberryPiFoundation/digital-editor-issues/issues/309)

### CODE REVIEW

File: src/components/Editor/ErrorMessage/ErrorMessage.jsx [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
+  const message = useRef();
   const error = useSelector((state) => state.editor.error);
   const settings = useContext(SettingsContext);
 
+  useEffect(() => {
+    if (message.current) {
+      message.current.innerHTML = error;
+    }
+  }, [error]);
+  console.log(error);
```
Comment (sra405) [inline src/components/Editor/ErrorMessage/ErrorMessage.jsx:16]: 🪓 
```suggestion
```

Comment (KristinaDudnyk) [reply src/components/Editor/ErrorMessage/ErrorMessage.jsx]: ready for the final approval )

**Review by sra405 — DISMISSED** [review]
👍 nice!!

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## Unreleased
 
+### Changed
+
+- Updated the ImportErrors message
+- In ErrorMassege component added the way to display html elements in string
```
Comment (loiswells97) [inline CHANGELOG.md:12]: Slight typo in `ErrorMessage`

File: src/components/Editor/Runners/PythonRunner/SkulptRunner/SkulptRunner.jsx [hunk 3]
```diff
@@ -304,8 +304,10 @@ const SkulptRunner = ({ active, outputPanels = ["text", "visual"] }) => {
       const fileName = err.traceback[0].filename.replace(/^\.\//, "");
 
       if (errorType === "ImportError" && window.crossOriginIsolated) {
+        const articleLink =
+          "https://help.editor.raspberrypi.org/hc/en-us/articles/30841379339924-What-Python-libraries-are-available-in-the-Code-Editor";
```
Comment (loiswells97) [inline src/components/Editor/Runners/PythonRunner/SkulptRunner/SkulptRunner.jsx:308]: This is probably fine for now, but I'm wondering whether in the future we might want to link this off to different locales with translations of the article 🤔

Comment (KristinaDudnyk) [reply src/components/Editor/Runners/PythonRunner/SkulptRunner/SkulptRunner.jsx]: Thanks @loiswells97 for you review!!
Locales were added

Comment (loiswells97) [reply src/components/Editor/Runners/PythonRunner/SkulptRunner/SkulptRunner.jsx]: This was just a thought for the future sorry, not something we want right now... I just checked and it looks like it's only the `en-us` locale that exists on the help site for now, `en-GB` for example gives a 404

**Review by loiswells97 — APPROVED** [review]
Looks great, nice one ✅

**Pull request merged**